troubleshooting/known_issues/SBPS_boot_fail.md

```diff
@@ -0,0 +1,114 @@
+# Boot content projection to Compute/UANs via iSCSI fails if the image does not have a `etag`
+
+## Issue Description
+
+iSCSI based boot content projection which is also known as "Scalable Boot Content Projection" (SBPS) for `rootfs` and `PE` images
+is supported in CSM 1.6.0. SBPS fails if the `rootfs` image manifest to be projected doesn't have a `etag` field or `etag` field

```

Comment:
```suggestion
is supported in CSM 1.6.0. SBPS fails if the `rootfs` image manifest to be projected doesn't have an `etag` field. 

```

I don't really understand what "doesn't have an `etag` field or `etag` field" is supposed to mean.   If something doesn't have an etag field we know it doesn't have an etag field, right?   Or am I missing something?